doc/client_lua_api.md

```diff
@@ -653,7 +611,7 @@ Helper functions
 * `table.copy(table)`: returns a table
     * returns a deep copy of `table`
 
-`minetest` namespace reference
+Minetest namespace reference
```

Comment:
Wasn't here `minetest` because the the table variable is named `minetest`?
You should add that you have to use `core.` instead `minetest.` yet.